pkg/apis/kops/componentconfig.go

```diff
@@ -461,5 +465,16 @@ func (c *KubeAPIServerConfig) HasAdmissionController(name string) bool {
 		}
 	}
 
+	for _, x := range c.DisableAdmissionPlugins {
```

Comment:
perhaps ...
```go
for _, x := range x.DisableAdmissionPlugins {
     if x == name { 
        return false
     }
}
for _, x := range append(c.AdmissionControl, c.EnableAdmissionPlugins...)` 
    if x == name { 
      return true // and save a few lines
    }
}
```
but up to you .. it's just code preference, so no biggie :-)

Reply:
Unless you and another reviewer feel strongly about combining them, I'd rather keep them separate. Subjectively, `range append(c.AdmissionControl, c.EnableAdmissionPlugins...)` is  harder to read, only to save a couple of lines. Presumably we'll be cleaning up `c.AdmissionControl` in the future anyway, when it has been removed from k/k and kops stops supporting older versions.